h2o-py/h2o/automl/autoh2o.py

```diff
@@ -654,3 +666,23 @@ def get_automl(project_name):
     :returns: A dictionary containing the project_name, leader model, leaderboard, event_log.
     """
     return H2OAutoML._fetch_state(project_name)
+
+
+def get_leaderboard(aml, extensions=None):
+    """
+    Retrieve the leaderboard from the AutoML instance.
+    Contrary to the default leaderboard attached to the automl instance, this one can return columns other than the metrics.
+    :param H2OAutoML aml: the instance for which to return the leaderboard.
+    :param extensions: a string or a list of string specifying which optional columns should be added to the leaderboard. Defaults to None.
+        Currently supported extensions are:
+        - 'ALL': adds all columns below.
```

Comment:
@seb-h2o I personally expect all string keys to be of the same case. Maybe all capitals?

Reply:
@seb-h2o but probably ALL is of a higher level semantic here

Reply:
'all' also works, but for clarity in the doc, I prefer making a distinction between a "normal" column name (like "training_time_ms") and a constant with a different semantic like "ALL".